eng/pipelines/common/templates/runtimes/build-test-job.yml

```diff
@@ -74,7 +74,7 @@ jobs:
       - ${{ each variable in parameters.variables }}:
         - ${{ variable }}
       - name: liveRuntimeBuildParams
-        value: 'libs.ref -c Release -ci'
+        value: 'libs.sfx+libs.oob /p:RefOnly=true -c Release -ci'
```

Comment:
Since there are concerns about removing `libs.ref` and having `/p:RefOnly=true` as the trigger to build the refs only, should we revert this change until a consensus is agreed on the issue open to discuss that?

Reply:
I would like the infrastructure to use the new subsets but keep the existing ones to not break existing developer workflows.